numpy/lib/tests/test_function_base.py

```diff
@@ -668,7 +668,7 @@ def test_ufunc(self):
         args = np.array([0, 0.5*np.pi, np.pi, 1.5*np.pi, 2*np.pi])
         r1 = f(args)
         r2 = np.cos(args)
-        assert_array_equal(r1, r2)
+        assert_array_almost_equal(r1, r2)
```

Comment:
Just a guess, but maybe it has to do with differences between the C standard libraries in MSVC vs the ones in MinGW. I can verify that they are different on my machine though. Only on the order of 1E-21 though.